CHANGELOG.markdown

```diff
@@ -1,3 +1,7 @@
+Next
```

Comment:
Can you put `[????.??.??]` after `Next`? That way, we'll remember to tag the next release with the date on which it was made.